package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "private": true,
+  "scripts": {
+    "clean": "rimraf output && rimraf .pulp-cache",
+    "build": "pulp build -- --censor-lib --strict",
+    "test": "pulp test -- --censor-lib"
+  },
+  "devDependencies": {
+    "pulp": "^15.0.0",
+    "purescript-psa": "^0.8.0",
+    "rimraf": "^3.0.2"
+  }
+}
```

Comment:
I think in the other contrib repositories we've just updated the CI.yml file directly, without introducing Pulp to the project (or a new package.json file). Can we just update the Bower file and the ci.yml file instead of adding these new files?

Reply:
In other words, the migration script for core / web / node isn't suitable for contrib.